src/platform/drivers/pl011_uart/inc/drivers/pl011_uart.h

```diff
@@ -1,210 +1,213 @@
+#define UART_RIS_CTSRMIS        (1 << 1)
+#define UART_RIS_DCDRMIS        (1 << 2)
+#define UART_RIS_DSRRMIS        (1 << 3)
+#define UART_RIS_RXRIS          (1 << 4)
+#define UART_RIS_TXRIS          (1 << 5)
+#define UART_RIS_RTRIS          (1 << 6)
+#define UART_RIS_FERIS          (1 << 7)
+#define UART_RIS_PERIS          (1 << 8)
+#define UART_RIS_BERIS          (1 << 9)
+#define UART_RIS_OERIS          (1 << 10)
+
+/* UART Masked Interrupt Status Register */
+
+#define UART_MIS_RIMMIS         (1 << 0)
+#define UART_MIS_CTSMMIS        (1 << 1)
+#define UART_MIS_DCDMMIS        (1 << 2)
+#define UART_MIS_DSRMMIS        (1 << 3)
+#define UART_MIS_RXMIS          (1 << 4)
+#define UART_MIS_TXMIS          (1 << 5)
+#define UART_MIS_RTMIS          (1 << 6)
+#define UART_MIS_FEMIS          (1 << 7)
+#define UART_MIS_PEMIS          (1 << 8)
+#define UART_MIS_BEMIS          (1 << 9)
+#define UART_MIS_OEMIS          (1 << 10)
+
+/* UART Interrupt Clear Register */
+
+#define UART_ICR_RIMIC          (1 << 0)
+#define UART_ICR_CTSMIC         (1 << 1)
+#define UART_ICR_DCDMIC         (1 << 2)
+#define UART_ICR_DSRMIC         (1 << 3)
+#define UART_ICR_RXIC           (1 << 4)
+#define UART_ICR_TXIC           (1 << 5)
+#define UART_ICR_RTIC           (1 << 6)
+#define UART_ICR_FEIC           (1 << 7)
+#define UART_ICR_PEIC           (1 << 8)
+#define UART_ICR_BEIC           (1 << 9)
+#define UART_ICR_OEIC           (1 << 10)
+
+/* UART DMA Control Register */
+
+#define UART_DMACR_RXDMAE       (1 << 0)
+#define UART_DMACR_TXDMAE       (1 << 1)
+#define UART_DMACR_DMAONERR     (1 << 2)
+
+/* For printk */
+
+#define serial_puts(str_buffer) uart_puts(1, str_buffer)
+
+/* UART (PL011) register structure */
+
+struct Pl011_Uart_hw {
+    volatile uint32_t data; // UART Data Register
+    volatile uint32_t status_error; // UART Receive Status Register/Error Clear
+                                    // Register
+    const uint32_t reserved1[4]; // Reserved: 4(0x4) bytes
+    volatile uint32_t flag; // UART Flag Register
+    const uint32_t reserved2[1]; // Reserved: 1(0x1) bytes
+    volatile uint32_t lp_counter; // UART Low-power Counter Register
+    volatile uint32_t integer_br; // UART Integer Baud Rate Register
+    volatile uint32_t fractional_br; // UART Fractional Baud Rate Register
+    volatile uint32_t line_control; // UART Line Control Register
+    volatile uint32_t control; // UART Control Register
+    volatile uint32_t isr_fifo_level_sel; // UART Interrupt FIFO level Select
+                                          // Register
+    volatile uint32_t isr_mask; // UART Interrupt Mask Set/Clear Register
+    volatile uint32_t raw_isr_status; // UART Raw Interrupt Status Register
+    volatile uint32_t masked_isr_status; // UART Masked Interrupt Status
+                                         // Register
+    volatile uint32_t isr_clear; // UART Interrupt Clear Register
+    volatile uint32_t DMA_control; // UART DMA control Register
```

Comment:
I would prefer this trailing comments to be aligned. Is this the AlignTrailingComments option?

Reply:
I've added this option. Please check if it formatted this as you expected.